pypeit/spectrographs/not_alfosc.py

```diff
@@ -111,21 +111,32 @@ def default_pypeit_par(cls):
         # Ignore PCA
         par['calibrations']['slitedges']['sync_predict'] = 'nearest'
         par['calibrations']['slitedges']['bound_detector'] = True
+        # Not a multi-slit instrument but flats are sometimes quite ugly leading to erroneous detection of multiple slits. Setting a higher edge_thresh seems to fail because of the bound_detector.
+        par['calibrations']['slitedges']['minimum_slit_gap'] = 15
 
         # Set pixel flat combination method
         par['calibrations']['pixelflatframe']['process']['combine'] = 'median'
         # Wavelength calibration methods
         #par['calibrations']['wavelengths']['method'] = 'holy-grail'
         par['calibrations']['wavelengths']['method'] = 'full_template'
-        par['calibrations']['wavelengths']['lamps'] = ['HeI', 'NeI']
+        par['calibrations']['wavelengths']['lamps'] = ['HeI', 'NeI', 'ArI', 'ArII']
```

Comment:
Are these lamps good for all the gratings or only the new ones?

Reply:
I believe the calibration plan is to provide single He, Ne and ThAr exposures for each setup used - http://www.not.iac.es/observing/seq/new/seq-command.php?i=alfosc.alfosc-calibs
- meaning they should be good for all grisms.  I intend to implement the rest of the grisms once I have access to the relevant data, so I'll submit a future pull request to make this a setup dependent parameter if I discover that isn't the case.